osf/models/schema_response_block.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+from osf.models.base import BaseModel, ObjectIDMixin
+
+
+class SchemaResponseBlock(ObjectIDMixin, BaseModel):
+
+    # Match length for RegistrationSchemaBlockKey.registration_response_key
+    schema_key = models.CharField(max_length=255)
+    answer = models.TextField()
```

Comment:
Should this be JSON? what if this is a file? Also I'd change this to `response`. For example if it's a file.

Reply:
good catch